Faizan_Akhtar/profile-page/assets/styles/profile-page.css

```diff
@@ -0,0 +1,11 @@
+.jumbotron{
+    background-image: linear-gradient(to right, #fe5858, #ee9617);  
+    /* colors - #2a2a72 #009ffd  */
+    /* #ee9617 , #fe5858  */
+}
+h1{
+    color:white !important;
```

Comment:
get rid of !important rule

Reply:
I'll give this h1 an id to get rid of !important rule